### PR TITLE
week2

### PR DESCRIPTION
**Pull request by mynewyear**
week2

### CODE REVIEW

File: 01-node-tutorial/answers/writeWithPromisesAwait.js
Comment (viniciuspietscher) [inline 01-node-tutorial/answers/writeWithPromisesAwait.js:1]: great job! 💯 🥇

File: 01-node-tutorial/answers/writeWithPromisesThen.js
Comment (viniciuspietscher) [inline 01-node-tutorial/answers/writeWithPromisesThen.js:1]: good job! 💯 🥇

**Review by viniciuspietscher — APPROVED** [review]
great job! 💯 🚀

File: 01-node-tutorial/answers/writeWithPromisesThen.js
Comment (mynewyear) [reply 01-node-tutorial/answers/writeWithPromisesThen.js:1]: thank you😀

**Pull request merged**